src/vm/comutilnative.cpp

```diff
@@ -911,6 +911,16 @@ FCIMPL5(void, GCInterface::GetMemoryInfo, UINT32* highMemLoadThreshold, UINT64*
 }
 FCIMPLEND
 
+FCIMPL1(INT64, GCInterface::GetTotalAllocatedBytes, CLR_BOOL precise)
+{
+    FCALL_CONTRACT;
+
+    FC_GC_POLL_NOT_NEEDED();
```

Comment:
What does that imply, especially since `precise = true` implies we will do a STW?

Reply:
`FC_GC_POLL_NOT_NEEDED` does not mean anything. It is unfinished scheme to make the GC suspension around FCalls to be more reliable.